public/components/feature-switches/feature-switches.js

```diff
@@ -54,10 +59,16 @@ function wfFeatureSwitchesController ($scope, wfPreferencesService) {
 
     const featureSwitches = new FeatureSwitches(getDefaultFeatureSwitchValues(), $scope.featureSwitchEntries)
 
-    const updateLocalFeatureSwitchValues = (featureSwitchResult) => {
-        const incomingFeatureSwitches = featureSwitchResult;
-        if (_.isObject(incomingFeatureSwitches)){
-            featureSwitches.update(incomingFeatureSwitches);
+    const updateLocalFeatureSwitchValues = (featureSwitchResult) => {   
+        const filteredIncomingFeatureSwitchResult = Object.keys(featureSwitchResult)
```

Comment:
This should mean that regardless of the preferences stored in the database, the only feature switches that get used when updating the UI are the ones that match to a current feature switch as defined by the `featureSwitchKeys` array.

Reply:
This seems to do the trick! 

![Screenshot 2024-06-12 at 15 41 53](https://github.com/guardian/workflow-frontend/assets/7423751/93c33c7c-59d7-4c07-a8e1-33f7fc85d328)